### PR TITLE
csv_to_excel

### LINKED ISSUE #2: Add csv to excel converter 🤖.

### PR DESCRIPTION
**Pull request by ayush-1909**
A python script to convert a file from csv to excel.

### CODE REVIEW

**Comment by mpotane** [comment]
> A python script to convert a file from csv to excel.

Have you signed up to hacktoberfest 2022 bro? [Hacktoberfest2022](https://hacktoberfest.com/)

**Review by mpotane — COMMENTED** [review]
Please add requirements.txt bro. Inside it is the module **openpyxl**

**Comment by ayush-1909** [comment]
> > A python script to convert a file from csv to excel.
> 
> Have you signed up to hacktoberfest 2022 bro? [Hacktoberfest2022](https://hacktoberfest.com/)

Yes I did! This repo is eligible right?

**Comment by ayush-1909** [comment]
> Please add requirements.txt bro. Inside it is the module **openpyxl**

Done, kindly have a look

File: csv_to_excel/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1. Python
```
Comment (mpotane) [inline csv_to_excel/requirements.txt]: remove python on line 1 bro.

**Review by ayush-1909 — COMMENTED** [review]
Changed everything as per requirements.

**Review by mpotane — APPROVED** [review]

**Review by mpotane — APPROVED** [review]

**Comment by mpotane** [comment]
> Changed everything as per requirements.

sorry bro i had to changed it to pandas since I want it to be much more beginner-friendly.

**Comment by ayush-1909** [comment]
> > Changed everything as per requirements.
> 
> sorry bro i had to changed it to pandas since I want it to be much more beginner-friendly.

Ah okay, no problem! Thanks a lot :)

**Comment by mpotane** [comment]
> > > Changed everything as per requirements.
> > 
> > 
> > sorry bro i had to changed it to pandas since I want it to be much more beginner-friendly.
> 
> Ah okay, no problem! Thanks a lot :)

I have opened two issues you can contribute very easily.

**Pull request merged**